ui/src/index.jsx

```diff
@@ -2,5 +2,8 @@ import React from 'react';
 import { render } from 'react-dom';
 import App from './App';
 import routes from './App/routes';
+import Link from './utils/Link';
+
+Link.setRoutes(routes);
 
 render(<App routes={routes} />, document.getElementById('root'));
```

Comment:
Now that we are using `Link.routes`, we could stop passing `routes` as a prop.

Reply:
Also a similar change should be done to `ui/src/docs.jsx`.

Reply:
This appears to still be used:
https://github.com/taskcluster/taskcluster/blob/1d97c53560ac1b20d53b3e79572a407b8996fc54/ui/src/App/index.jsx#L203-L214
https://github.com/taskcluster/taskcluster/blob/1d97c53560ac1b20d53b3e79572a407b8996fc54/ui/src/App/Main.jsx#L95-L106

Reply:
Not sure what I was thinking when I suggested this. You are right.

I think https://github.com/taskcluster/taskcluster/pull/1312#discussion_r326192504 still needs to be done however.

Reply:
Good point -- done.